urllib3/_collections.py

```diff
@@ -38,27 +73,32 @@ class RecentlyUsedContainer(MutableMapping):
         ``dispose_func(value)`` is called.  Callback which will get called
     """
 
-    ContainerCls = OrderedDict
+    # TODO be more precise about class type
+    ContainerCls = OrderedDict  # type: type
 
     def __init__(self, maxsize=10, dispose_func=None):
+        # type: (int, Optional[Callable[[_V], None]]) -> None
         self._maxsize = maxsize
         self.dispose_func = dispose_func
 
-        self._container = self.ContainerCls()
+        self._container = self.ContainerCls()  # type: OrderedDict[_K, _V]
```

Comment:
Yup, because using `popitem(last=...)` which is only defined by `OrderedDict` (in this case, why having a `ContainerCls`?).